procedures/unit-testing-constants.ipf

```diff
@@ -137,3 +138,12 @@ StrConstant UTF_KEY_REGEXP               = "REGEXP"
 /// @{
 Constant UTF_MAX_PROC_LINES = 10000
 /// @}
+
+/// @name Wave tracking modes. These are flags that can be combined
```

Comment:
"These are flags that can be combined" that is not fully correct, as you would have then only 1 and 2 as constant. What you mean is that the tracking modes are a bit pattern.
I personally tend to use for bits the hex notation through
0x01 and 0x02 etc. as a kind of visual differentiation to enums.